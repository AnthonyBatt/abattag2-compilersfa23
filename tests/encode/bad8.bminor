"This is a nonescaped quote: " see how it causes an error?"
