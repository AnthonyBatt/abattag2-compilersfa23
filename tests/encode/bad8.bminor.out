An " must be at the start or end of the string, if you meant to print one then please escape it: \"
decode("This is a nonescaped quote: " see how it causes an error?"): the string was unable to be decoded
